storage/boltdb/client_test.go

```diff
@@ -28,6 +30,7 @@ func TestSuite(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to create db: %v", err)
 	}
+	store.db.MaxBatchDelay = 1 * time.Millisecond
```

Comment:
It's a little worrying anytime we have to run the test environment with a configuration that is so different from what is in prod. Are there any ways to avoid it without entirely breaking test performance? Maybe doing more test operations at the same time, so that batches would naturally tend to complete more quickly?

Reply:
yeah i can remove this if its not ideal for testing.